src/App.jsx

```diff
@@ -1,18 +1,50 @@
+import { useState } from 'react';
 import './App.css';
 import LeafletMap from './components/leaflet/LeafletMap';
 import Header from './components/header/Header';
-const isLoading = false;
+import { fetchLocation } from './helpers/fetchLocation';
+import { transformGeoInfo } from './helpers/transformGeoInfo';
+
+const initialGeoInfo = {
+  ip: '',
+  isp: '',
+  country: '',
+  timezone: '',
+  coordinates: [51.505, -0.09],
```

Comment:
We can prefill all fields, not only coordinates.

Reply:
1 don’t  understand for whet?
2 if i do it i can’t  have '-' like default status because it will be lake that :
![Uploading Снимок экрана 2023-12-16 в 13.44.54.png…]()

Reply:
![Uploading Снимок экрана 2023-12-16 в 13.44.54.png…]()

Reply:
it's strange: image doesn't shown

Reply:
Okay. If you have a purpose for this to be blank then okay. Let's leave it as it is.